### PR TITLE
Add a TLDR and roadmap, plus info for contributors

### PR DESCRIPTION
**Pull request by Hex-4**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "cSpell.words": [
+        "purplebubble"
+    ]
+}
```
Comment (jaspermayone) [inline .vscode/settings.json:5]: Please don't commit this file.

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+This is a list of what the Purple Bubble project has done, and what still needs to be done.
+
+> **Contributors:** Feel free to help out with any of these tasks! The Purple Bubble team mostly communicates via #purplebubble in the [Hack Club Slack](hack.club/slack). (note: age verification is currently required to join.) If you can't join or would prefer not to, reach out to team@purplebubble.org or open a github issue if you have questions or need help.
+>
```
Comment (jaspermayone) [inline ROADMAP.md:4]: Please update text. 

> The Purple Bubble team & community communicates via slack! You can join the slack at https://join.slack.com/t/thepurplebubble/signup.

File: TLDR.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# TLDR for the TLDR
```
Comment (jaspermayone) [inline TLDR.md:1]: This title should be something like "Protocol TLDR"

File: TLDR.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+We're building a secure, decentralized communication protocol. Protocol v1.0.0 Jasper has multiple REST API servers in a mesh network. Clients connect to different TX (transmission) and RX (reception) servers. Servers can detect and report misbehaving servers. Everything is signed by users' keys.
```
Comment (jaspermayone) [inline TLDR.md:3]: @Luke-Oldenburg can you review here plz

File: TLDR.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+# Introduction
+
+Hey! This is Purple Bubble. We're a crew of teens on a mission to make messaging epic again. Zero tracking, spying, or data selling. Everything is end-to-end encrypted, anonymized, and decentralized. All open source.
+
+> **Wanna chat?**
+>
+> We mostly communicate through #purplebubble in the [Hack Club Slack](hack.club/slack), but since Hack Club is running a (super cool, you should check it out if you're a teen or younger) [summer event](hack.club/arcade), you will need to verify that you're not an adult to join. Worry not! You can always reach out to team@purplebubble.org or open a github issue.
```
Comment (jaspermayone) [inline TLDR.md:11]: Can you update this to use the purple bubble slack and remove hack club references pls

File: TLDR.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+
+# Why
+
+These days, no one can really trust tech companies. Slack has [started training AI models on messages](https://arstechnica.com/tech-policy/2024/05/slack-defends-default-opt-in-for-ai-training-on-chats-amid-user-outrage/), Discord is adding [ads to their app](https://arstechnica.com/gadgets/2024/04/discord-starts-down-the-dangerous-road-of-ads-this-week/).
```
Comment (jaspermayone) [inline TLDR.md:15]: let's remove this section.

**Review by jaspermayone — CHANGES_REQUESTED** [review]

**Comment by jaspermayone** [comment]
@Luke-Oldenburg would love it if you could give @Hex-4 feedback on the section I @mentioned you on

File: TLDR.md (hunk 4 above)
Comment (Luke-Oldenburg) [reply TLDR.md:3]: No server misbehavior detection yet. TX and RX servers are identical. Clients will only hit specific endpoints from each one they choose.

**Comment by jaspermayone** [comment]
@Hex-4 can you move this PR to https://github.com/thepurplebubble/meta plz

**Pull request closed without merging**